src/modules/Accordion/Accordion.js

```diff
@@ -101,24 +101,28 @@ export default class Accordion extends Component {
   renderChildren = () => {
     const { children } = this.props
     const { activeIndex } = this.state
+    let titleIndex = 0
+    let contentIndex = 0
 
-    return Children.map(children, (child, i) => {
+    return Children.map(children, (child) => {
       const isTitle = child.type === AccordionTitle
       const isContent = child.type === AccordionContent
 
       if (isTitle) {
-        const isActive = _.has(child, 'props.active') ? child.props.active : activeIndex === i
+        const currentIndex = titleIndex
```

Comment:
Could we remove `currentIndex` from this block and the isContent block and just use `titleIndex` and `contentIndex` respectively?  I think it is a little more clear.